### DISCUSSION TITLE
Github Desktop / Nasuni server error when committing

### DISCUSSION THREAD

**Discussion by Rob112233** [community/community#112280] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've been using GitHub Desktop for several years to connect to a repository that is hosted on one of our supplier's servers in the cloud. I'm not very tech savvy when it comes to Github, but I was able to clone the repository from the cloud to our local server and we've been utilizing it for several years without any issues, until a few weeks ago.

My corporate IT department made a change to our servers to go from our local Windows servers to remote Nasuni cloud servers, they migrated all of our data over so it all looks the same as it did, but I've not been able to commit since the server migration. Our IT has not been able to figure out what is wrong and has tried getting Nasuni reps to help track down the issue but now they are turning to me to try to get ahold of Github Desktop support to try and put a call together, but I'm not really seeing a way to do that, so thought I would try the forums...

The issue I get is a fatal: Unable to write new index file error. 
![image](https://github.com/community/community/assets/161346180/0450364f-100f-4a74-bb36-e804e0c2c610)

I apologize in advance, I'm not a power Git users in any sense... all we use GitHub Desktop for is to connect to the supplier's servers and upload simple rtf / pdf files which are then displayed inside of our program. I don't really even know where to start with trying to ask for help.

I was able to clone the repository to my local C drive and everything works as it should... but trying to use the repository stored on the Nasuni server is extremely slow and then gives me the error writing index file. We have multiple users that had been using this shared server folder instead of cloning the repository to each person's computer, and they are resistant to going to using the local clone and continue to push me to try to find a solution.. is anyone able to point me towards any known issues with Nasuni servers... linux issues? Ask me questions to guide me what I need to know to be able to provide what is needed... sorry I am very lost.

I did find an issue with this error message stating that it could be related to a cloud sync issue if there is a monitor in place, but my IT guy says that's not the issue. Is there a way to get a Github Desktop person on the phone with my IT guy so they can talk IT guy to IT guy and I'm not playing middle man?

**Comment by pkkid** [comment]
Did you clone a Github repo to a shared network directory? It's pretty unusual to be working on a git repo thats on a shared network drive. Is it possible for you to move your local cloned repo off the Nasuni network share and on onto another directory on your PC? If its a git repo, the code is presumably already backed up wherever you cloned it from, thus there would be no need to be working out of a shared network directory.

**Reply by Rob112233** [reply]
Originally it was in a network drive that was not being backed up on a regular basis, they moved all of our local windows servers over to cloud Nasuni (linux) servers which are apparently backed up all the time. If there aren't known issues that are easily resolved with the Nasuni servers then I'm going to push them to just give us a local server that isn't backed up again. We're not using git for the repo server, but the server we are using for git does get backed up regularly, so I've attempted to communicate that to our IT guys that we don't need them backing the files up.

I'm currently running the cloned repo on my C drive which is working fine, but I have a few people on my team who are not tech oriented and are afraid of having multiple cloned repo's rather than how we've just had one that everyone used for the last several years... despite it only being one additional step to pull from origin to sync everything.
